hudi-common/src/main/java/org/apache/hudi/common/table/timeline/HoodieInstant.java

```diff
@@ -136,10 +136,7 @@ public HoodieInstant(StoragePathInfo pathInfo) {
           state = State.COMPLETED;
         }
       }
-      completionTime = timestamps.length > 1
-          ? timestamps[1]
-          // for backward compatibility with 0.x release.
-          : state == State.COMPLETED ? pathInfo.getModificationTime() + "" : null;
+      completionTime = timestamps.length > 1 ? timestamps[1] : null;
```

Comment:
I think we still need to keep these logic, a downgrade logic in there is good but we also need to be compatible for 0.x for some code path.